app/assets/images/corona-landing-page-logos.svg

```diff
@@ -0,0 +1,61 @@
+<svg width="476" height="61" viewBox="0 0 476 61" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Just out of curiousity how big is this SVG? The PNG was hi res and came in at 9.17 KB.

Reply:
> SVG doesn't work in IE8. Not sure if that's a problem or worth putting some effort into a fallback.

Good point - I'll look into that.



> Just out of curiousity how big is this SVG? The PNG was hi res and came in at 9.17 KB.

I think there's just more code in the vector version